projects/rero/ng-core/src/lib/record/editor/extensions.ts

```diff
@@ -74,11 +74,15 @@ export function getKey(field: any): string {
   }
 
   if (parentKey != null) {
-    let newId = parentKey;
+    const keyParams = [];
     if (field.key != null) {
-      newId = [parentKey, field.key].join('-');
+      keyParams.push(parentKey);
+      keyParams.push(field.key);
     }
-    return newId;
+    if (field.id) {
```

Comment:
I do not understand as the idea is to generate the id.

Reply:
done